props/saber_sabersense_buttons.h

```diff
@@ -0,0 +1,1046 @@
+        Off();
+          last_twist_ = millis();
+          saber_off_time_ = millis();
+          battle_mode_ = false;
+        }
+      return true;
+#endif
+
+#ifdef SABERSENSE_STAB_ON
+    case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_OFF):
+      if (millis() - saber_off_time_ > 1000) {
+        FastOn();
+        //  On();
+      }
+    return true;
+#endif
+
+#ifdef SABERSENSE_THRUST_ON
+    case EVENTID(BUTTON_NONE, EVENT_THRUST, MODE_OFF):
+      if (millis() - saber_off_time_ > 1000) {
+        FastOn();
+        //  On();
+      }
+    return true;
+#endif
+
+  //  Multiple Skips only available with 2 button installs.
+  //  Skips forward five fonts if pointing up, skips back five fonts if pointing down.
+    case EVENTID(BUTTON_AUX, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+      // backwards if pointing down
+      SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -5 : 5), true);
+    #ifdef SAVE_PRESET
+      SaveState(current_preset_.preset_num);
+    #endif
+    return true;
+
+  // Skips forward ten fonts if pointing up, skips back ten fonts if pointing down.
+    case EVENTID(BUTTON_AUX, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
+      // backwards if pointing down
+        SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -10 : 10), true);
+      #ifdef SAVE_PRESET
+        SaveState(current_preset_.preset_num);
+      #endif
+    return true;
+
+// Saber ON AND Volume Adjust.
+    case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+      IgnoreClash(50);  //  Hopefully prevents false clashes due to 'clicky' button.
+                        //  Low threshold so as not to conflict with 1-button lockup or volume menu.
+        if (!mode_volume_) {
+          On();
+        } else {
+          if (fusor.angle1() > 0) {
+            VolumeUp();
+          } else {
+            VolumeDown();
+        }
+      }
+    return true;
+
+  // Modified 2 button 'next/previous preset' AND volume down,
+  // to align with multiple skips above.
+  // Forward one font pointing up, back one font pointing down.
+    case EVENTID(BUTTON_AUX, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+      // backwards if pointing down
+      if (!mode_volume_) {
+        SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -1 : 1), true);
+      } else {
+        VolumeDown();
+      }
+#ifdef SAVE_PRESET
+      SaveState(current_preset_.preset_num);
+#endif
+    return true;
+
+  // 1 button 'next/previous preset' command.
+  // Forward one font pointing up, back one font pointing down.
+#if NUM_BUTTONS == 1 
+    case EVENTID(BUTTON_POWER, EVENT_SECOND_HELD, MODE_OFF):
+      // backwards if pointing down
+      if (!mode_volume_) {
+        SetPreset(current_preset_.preset_num + (fusor.angle1() < -M_PI / 4 ? -1 : 1), true);
+     } else {
+        VolumeDown();
+     }
+#ifdef SAVE_PRESET
+      SaveState(current_preset_.preset_num);
+#endif
+  return true;
```

Comment:
Why isn't this return statement indented properly (should match previous line generally)

Reply:
Sorted.